.drone.yml

```diff
@@ -0,0 +1,157 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/notifications
+
+branches: [master]
```

Comment:
recommend `branches: [master, release*]`

Reply:
there are no `release*` branches in this repo. So would that use the tags, or should we use `stable*`?

Reply:
since we are moving forward with unbundling also more and more apps - it will eventually have a `release` branch